design/request_management.puml

```diff
@@ -0,0 +1,92 @@
+@startuml
```

Comment:
Please add a png version of the diagram as well

Reply:
ok